src/required_resources.h

```diff
@@ -83,13 +88,22 @@ struct MapInfo {
   std::optional<LinearHistogramArgs> lhist_args;
   std::optional<int> hist_bits_arg;
   int id = -1;
+  int max_entries = -1;
+  bool has_decl_bpf_type = false;
```

Comment:
What is the significance of this? It is not serialized. I would argue that this class should hold the info for serialization purposes only, so it's a bit of a flag that there's something here that's not being serialized. (Maybe this is an accidental omission, or maybe the class is being used for multiple purposes)

Reply:
Leftover from an older version. Will remove. Good catch.